code/datums/components/crafting/recipes/recipes_clothing.dm

```diff
@@ -130,7 +156,6 @@
 	reqs = list(/obj/item/stack/sheet/durathread = 7,
 				/obj/item/stack/sheet/leather = 3)
 	time = 70
-	always_availible = TRUE
 	category = CAT_CLOTHING
```

Comment:
Broken commenting on Stunglasses. Fix this.

Reply:
?

Reply:
You broke where Stunglasses were commented out.